src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
   other than the three types mentioned.
   ([#4542](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4542))
 
+* Added [Logs Bridge
+  API](https://github.com/open-telemetry/opentelemetry-specification/blob/976432b74c565e8a84af3570e9b82cb95e1d844c/specification/logs/bridge-api.md)
+  artifacts.
```

Comment:
What does "artifact" mean here? (which part of https://en.wikipedia.org/wiki/Artifact_(software_development))

Reply:
"Things" 🤣 I changed the entry to call out the main pieces LMK if you want it to say more.

Reply:
I guess many folks would interpret "adding ... artifacts" as "introducing a new NuGet package or DLL". I suggest that we simply say "added logs bridge API" ("to the existing API artifact" - which can be easily deduced from the changelog context).

Reply:
Something weird is going on with GitHub. I pushed to the branch behind this PR but the PR isn't updating. Here's the commit if you want to check out the text I'm _trying_ to use 😄 

https://github.com/CodeBlanch/opentelemetry-dotnet/commit/20e0221133ff2f164adc71cd612bb66e48ace56b

Reply:
The new version is much easier to understand 👍